css/properties/inset-block-end.json

```diff
@@ -84,24 +89,29 @@
                 ]
               }
             ],
-            "opera_android": {
-              "version_added": "48",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "enable-experimental-web-platform-features",
-                  "value_to_set": "enabled"
-                }
-              ]
-            },
+            "opera_android": [
+              {
+                "version_added": "62"
+              },
+              {
+                "version_added": "48",
```

Comment:
Opera on Android doesn't really support flags (we can't formally block them yet; see https://github.com/mdn/browser-compat-data/issues/7354, https://github.com/mdn/browser-compat-data/issues/10616). Would you mind dropping the flagged support statements from all the Opera for Android changes in this PR?